sdk/monitor/azquery/cloud_config.go

```diff
@@ -0,0 +1,39 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azquery
+
+import "github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
+
+const monitorQueryLogs cloud.ServiceName = "azqueryLogs"
```

Comment:
Nit, these should go in a `const` block.
```go
const(
    // put constants here
)
```

Reply:
fixed in commit [d2eafa6](https://github.com/Azure/azure-sdk-for-go/pull/19462/commits/d2eafa65677cd3eeccdfa1f740ecf1ff6bd959f9)